### PR TITLE
Step 3: Mobile meunu

### PR DESCRIPTION
**Pull request by leehaney254**
## In this system 👍 
- Added a mobile menu
- Used JavaScript to toggle the menu
- Modified the readme.md file

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -7,10 +7,36 @@ label {
...
+#nav-link {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  z-index: 99;
+  background: #3c3a39;
+  transition: transform 0.1s;
+}
```
Comment (Gopxfs) [inline styles.css:23]: Nice job with your menu functionality! Please make sure to follow the same design displayed in Figma 😉.
| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

Kindly notice:
- The links should begin with a capital letter (Portfolio & Contact).
- In your project, it is misspelled "portolio" 😉
- In the original design, the lines under the name of the links are wider, with the name of the links starting after some spacing:
image
- The close button ('X') has more spacing between itself and the right border:

| Figma design  | Your design |
| ------------- | ------------- |
| image  | image  |

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.

**Review by Gopxfs — CHANGES_REQUESTED** [review]
# ♻️ STATUS: REQUIRED CHANGES ♻️

## Hi @leehaney254 @solorics 👋,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights :rocket:
✔️ No linter errors.
✔️ Great code structure. Well done!
✔️ Menu working as intended.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 🗒️  
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Gopxfs in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leehaney254** [comment]
## Changes 💪 
**I have:**
- Capitalized the first letter of the links
- Changed the spelling of the Portfolio
- Created wider lines at the bottom
- Spaced the close button

**Review by gemmen29 — APPROVED** [review]
Hi @leehaney254, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ]  [OPTIONAL] Kindly use the JavaScript + HTML + CSS linter as per this [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md#:~:text=For%20this%20project%2C%20you%20should%20add%20the%20JavaScript%20%2B%20HTML%20%2B%20CSS%20linter.). Now the js linters is not running. You need to change the linters.yml file with the new one in order to run. This will mark as optional as the previous reviewer does not request it but in the upcoming project, the PR will mark as invalid.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**